### PR TITLE
Frontend/bugfixes/iss300

### PR DESCRIPTION
**Pull request by Rintel**

### CODE REVIEW

**Review by markwitz — DISMISSED** [review]
rebase plz to not drop backend code

**Comment by mharasic** [comment]
FINISHED

File: tests/api/conftest.py [hunk 1]
```diff
@@ -90,10 +90,10 @@ def enter_data_using_post(pytestconfig, session):
...
 def enter_archived_using_post(pytestconfig, flask_api_url, session):
     test_manifest = os.path.join(
-     str(pytestconfig.rootdir),
-     'tests',
-     'testmanifests',
-     'validexample0_archived.json5'
+        str(pytestconfig.rootdir),
+        'tests',
+        'testmanifests',
+        'validexample0_archived.json5'
```
Comment (henderkes) [inline tests/api/conftest.py]: this was already fixed in the corresponding branch, but whatever

**Review by henderkes — COMMENTED** [review]
I have zero knowledge about the frontend or javascript at all, so sorry for the short review.

File: tests/api/conftest.py (hunk 1 above)
Comment (mharasic) [reply tests/api/conftest.py]: Came through dirty merge... only a single line

**Comment by Rintel** [comment]
As I'm the author of the Pull Request, I can't approve it. Therefore one of you should please do it, @Drakulix is waiting for the deployment.

**Review by mharasic — DISMISSED** [review]
Sounds working?

File: web/src/components/views/CreateProjectFromFile.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            if(data.date_creation!=null){
+              var projectDate = new Date(data.date_creation)
+              this.setState({date : projectDate})
+              project["date_creation"] = data.date_creation
+            }
+            if(data.description!=null){
+              project["description"] = data.description
+            }
+            if(data.status!=null){
+              switch(data.status.toLowerCase()){
```
Comment (mharasic) [inline web/src/components/views/CreateProjectFromFile.jsx]: status ist mitlerweile analog zum file definiert und ist nicht mehr zahlen basiert

File: web/src/components/views/CreateProjectFromFile.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+              var projectAuthors = []
+              for(var i = 0;i<data.authors.length;i++){
+                  projectAuthors.push(data.authors[i].name + " ("+data.authors[i].email + ")")
+              }
+              this.setState({ authors : projectAuthors })
+            }
+            if(data.date_creation!=null){
+              var projectDate = new Date(data.date_creation)
+              this.setState({date : projectDate})
+              project["date_creation"] = data.date_creation
```
Comment (mharasic) [inline web/src/components/views/CreateProjectFromFile.jsx:43]: date_creation is okay.... but ...
date for the datepicker has to be defined as well

**Review by mharasic — CHANGES_REQUESTED** [review]
See inline comments for 

https://github.com/Drakulix/knex/pull/310/commits/bca2ecb65d9a9ab36d816f57b169baba9dce5220

File: web/src/components/views/CreateProjectFromFile.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+const JSON5 = require('json5')
+
+export default class CreateProjectByURL extends React.Component {
+
+      constructor(props) {
+        super(props)
+        var data = this.props.json;
+        this.state = {
+          status : "2",
```
Comment (mharasic) [inline web/src/components/views/CreateProjectFromFile.jsx]: Status is now sting based DONE; IN_PROGRESS:...

**Review by mharasic — CHANGES_REQUESTED** [review]

**Review by mharasic — APPROVED** [review]

**Pull request merged**